src/app/core/services/rundown-state.service.ts

```diff
@@ -83,7 +83,7 @@ export class RundownStateService implements OnDestroy {
       this.rundownEventObserver.subscribeToRundownReset(this.resetRundownFromEvent.bind(this)),
       this.rundownEventObserver.subscribeToRundownTake(this.takePartInRundownFromEvent.bind(this)),
       this.rundownEventObserver.subscribeToRundownSetNext(this.setNextPartInRundownFromEvent.bind(this)),
-      this.rundownEventObserver.subscribeToRundownInfinitePieceAdded(this.addInfinitePieceToRundownFromEvent.bind(this)),
+      this.rundownEventObserver.subscribeToRundownInfinitePieceAdded(this.updateInfinitePiecesFromEvent.bind(this)),
```

Comment:
`.subscribeToRundownInfinitePieceAdded` should be updated too

Reply:
Updated!